src/contracts.rs

```diff
@@ -583,3 +607,339 @@ impl WatchOnlySwapCoin {
         })
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use bitcoin::consensus::encode::deserialize;
+    use bitcoin::hashes::hex::{FromHex, ToHex};
+    use bitcoin::PrivateKey;
+    use rand::{thread_rng, Rng};
+    use std::str::FromStr;
+    use std::string::String;
+
+    #[test]
+    fn test_maker_pubkey_computation() {
+        let secp = Secp256k1::new();
+        let sk =
+            PrivateKey::from_wif("cVt4o7BGAig1UXywgGSmARhxMdzP5qvQsxKkSsc1XEkw3tDTQFpy").unwrap();
+
```

Comment:
These functions have an extra line break after every line, that's not necessary and should be removed.

Reply:
This lint level formatting is done by rustfmt. So its being automatically created by the fmt run.